[change] Documentation update (#4)
trueos/build: README.md

3. Do nothing and automatically use the default build manifest (trueos-snapshot). 
   * NOTE: The trueos-snapshot manifest only builds a tiny subset of packages and is primarily used for testing the build procedures and base OS packages only.

# Usage
Standard commands to build a TrueOS distribution:
```
# make ports
# make iso
```

## All Options

### make clean
This command will clean up files from previous builds. Running this command is typically not required by the user, as the individual cleaning operations are dynamically run as needed during a build procedure so that the current build is not negatively impacted by previous build artifacts.

This command will clean up:

* Poudriere jails, ports trees, and mountpoints.
* ISO output directory : Deleting the output of any "make iso" commands run previously.
* VM output directory : Deleting the output of any "make vm" commands run previously.

### make config
This will launch an interactive prompt to select a build manifest from the example files in the [manifests directory](https://github.com/trueos/build/tree/master/manifests) and use that as the default build manifest.

> This selection will get saved to the local ".config/manifest" file and used whenever the **TRUEOS_MANIFEST** environment variable is not set. This command may be run whenever a change your default manifest is desired.

Reviewer: ```suggestion
The selected Manifest name will get saved to the local ".config/manifest" file and used whenever the **TRUEOS_MANIFEST** environment variable is not set. This command may be run whenever a different default manifest is desired.
```

**WARNING:** If no build manifest is specified (either by running `make config` or providing the TRUEOS_MANIFEST environment variable), then the "trueos-snapshot.json" build manifest will be used automatically from the [manifests directory](https://github.com/trueos/build/tree/master/manifests).

### make ports

> Build all of the OS and ports and assemble a package repository.

Reviewer: ```suggestion
This will build the OS and ports followed by assembling a package repository.
```

Ok Don't love second half of my suggestion but feel somethings needed here.

Author: I just adjusted the phrasing in that section a bit more.

# Usage
Standard commands to build a TrueOS distribution:
```
# make ports
# make iso
```

## All Options

### make clean
This command will clean up files from previous builds. Running this command is typically not required by the user, as the individual cleaning operations are dynamically run as needed during a build procedure so that the current build is not negatively impacted by previous build artifacts.

This command will clean up:

* Poudriere jails, ports trees, and mountpoints.
* ISO output directory : Deleting the output of any "make iso" commands run previously.
* VM output directory : Deleting the output of any "make vm" commands run previously.

### make config
This will launch an interactive prompt to select a build manifest from the example files in the [manifests directory](https://github.com/trueos/build/tree/master/manifests) and use that as the default build manifest.
The selected Manifest name will get saved to the local ".config/manifest" file and used whenever the **TRUEOS_MANIFEST** environment variable is not set. This command may be run whenever a different default manifest is desired.

> **WARNING:** If no build manifest is specified (either by running `make config` or providing the TRUEOS_MANIFEST environment variable), then the "trueos-snapshot.json" build manifest will be used automatically from the [manifests directory](https://github.com/trueos/build/tree/master/manifests).

Reviewer 2: Its trueos-snapshot-builder.json now.

Author: Fixed.
Apparently the build.sh script was never updated to look for the "-builder" manifest by default either - so I went ahead and fixes that in build.sh as well.
